### PR TITLE
Fix for recursive directories causing an infinite loop

### PR DESCRIPTION
**Pull request by danielbardsley**
Ensure we don't get stuck in an endless loop when uploading recursive directories: https://github.com/gilmarsquinelato/serverless-spa/issues/1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -375,9 +375,11 @@ class SPA {
     return this.aws.request('S3', 'putBucketPolicy', params, this.stage, this.region);
   }
 
-  _uploadDirectory() {
-    const directoryPath = this.distFolder;
-    this.serverless.cli.log('Uploading files...');
```
Comment (bzemms) [inline index.js]: Why did you remove this log?

Comment (gilmarsquinelato) [reply index.js]: New version v1.2.4 with that log back
